chainer/__init__.py

```diff
@@ -4,8 +4,8 @@
 import warnings
 
 from chainer import _version
+from chainer import backends  # NOQA
```

Comment:
I think it is better to fix following two lines too.

```
from chainer.cuda import should_use_cudnn  # NOQA
from chainer.cuda import should_use_cudnn_tensor_core  # NOQA
```

Reply:
Thank you. Fixed.